packages/cli/src/commands/config.ts

```diff
@@ -57,7 +57,7 @@ export default class ConfigCommand extends SmartThingsCommand {
 		]
 		const tableFieldDefinitions: TableFieldDefinition<ConfigItem>[] = [
 			...listTableFieldDefinitions,
-			{ label: 'Definition', value: (item) => yaml.safeDump(item.data) },
+			{ label: 'Definition', value: (item) => yaml.dump(item.data) },
```

Comment:
All of these are equivalently "safe". See [migration-guide-from-js-yaml3-to-js-yaml4](https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md#migration-guide-from-js-yaml3-to-js-yaml4)